.config/config.toml

```diff
@@ -1,4 +1,3 @@
-[capture]
```

Comment:
Oooh spicy. I dont expect we could do such a thing.

Reply:
Now we have a full power to do such a things